src/registry.jl

```diff
@@ -129,17 +129,59 @@ add_saver(format"WPG", :ImageMagick)
 # Mesh formats
 
 add_format(format"OBJ", (), ".obj")
-add_loader(format"OBJ", :WavefrontObj)
-add_saver(format"OBJ", :WavefrontObj)
+add_loader(format"OBJ", :MeshIO)
```

Comment:
Not that it much matters, but originally I had this file organized into "simple cases" (no function needed) at the top, and "complex cases" at the bottom. They are now intermingled; might be best to delete those comments.